suite-native/formatters/src/components/CryptoToFiatAmountFormatter.tsx

```diff
@@ -34,7 +29,5 @@ export const CryptoToFiatAmountFormatter = ({
         cryptoValue: value,
     });
 
-    const formattedFiatValue = FiatAmountFormatter.format(fiatValue ?? '0');
-
-    return <AmountText value={formattedFiatValue} isDiscreetText={isDiscreetText} {...textProps} />;
+    return <FiatAmountFormatter network={network} value={fiatValue} {...textProps} />;
```

Comment:
why to exclude `isDiscreetText`?

Reply:
It's not used anywhere with this component and the default is the same as in `FiatAmountFormatter`. It's practically dead code. ☠️

Reply:
makes sense 👍